packages/synthetic-chain/src/cli/dockerfileGen.ts

```diff
@@ -242,6 +243,12 @@ export function writeDockerfile(
     blocks.push(stage.TEST(proposal));
     previousProposal = proposal;
   }
+  // If one of the proposals is a passed proposal, make the latest one the default entrypoint
+  const lastPassed = lastPassedProposal(allProposals);
+  if (lastPassed) {
+    blocks.push(stage.DEFAULT(lastPassed));
+  }
```

Comment:
Confirmed equivalent of https://github.com/Agoric/agoric-3-proposals/pull/46/files#diff-f205598be3b5c05a2daf94abe7dcfb6231f6a0dc08814bbc47a5eae016d8be8aL207